scripts/apps/search/controllers/SearchMenuController.ts

```diff
@@ -56,7 +57,7 @@ export default function SearchMenuController(
 
     const initActiveProvider = () => {
         if ($location.path() === '/search') {
-            if (!$location.search()._id) {
+            if (!$location.search()._id && !$location.search().item) { // prevent from changing active provider
```

Comment:
It would help more if the comment answered questions "why?" and "under what circumstances?"